adapters/minutemedia/minutemediatest/exemplary/banner-and-video-app.json

```diff
@@ -0,0 +1,211 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            },
+            {
+              "w": 300,
+              "h": 300
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "org": "72721",
+            "path": "mvo",
+            "zone": "1r"
+          }
+        }
+      },
+      {
+        "id": "test-imp-video-id",
+        "video": {
+          "mimes": ["video/mp4"],
+          "minduration": 1,
+          "maxduration": 2,
+          "protocols": [1, 2, 5],
+          "w": 1020,
+          "h": 780,
+          "startdelay": 1,
+          "placement": 1,
+          "playbackmethod": [2],
+          "delivery": [1],
+          "api": [1, 2, 3, 4]
+        },
+        "ext": {
+          "bidder": {
+            "org": "72721",
+            "path": "mvo",
+            "zone": "1r"
+          }
+        }
+      }
+    ],
+    "app": {
+      "id": "agltb3B1Yi1pbmNyDAsSA0FwcBiJkfIUDA",
+      "name": "Yahoo Weather",
+      "bundle": "12345",
+      "storeurl": "https://itunes.apple.com/id628677149",
+      "cat": ["IAB15", "IAB15-10"],
+      "ver": "1.0.2",
+      "publisher": {
+        "id": "1"
+      }
+    },
+    "device": {
+      "dnt": 0,
+      "ua": "Mozilla/5.0 (iPhone; CPU iPhone OS 6_1 like Mac OS X) AppleWebKit / 534.46(KHTML, like Gecko) Version / 5.1 Mobile / 9 A334 Safari / 7534.48 .3",
+      "ip": "123.145.167.189",
+      "ifa": "AA000DFE74168477C70D291f574D344790E0BB11",
+      "carrier": "VERIZON",
+      "language": "en",
+      "make": "Apple",
+      "model": "iPhone",
+      "os": "iOS",
+      "osv": "6.1",
+      "js": 1,
+      "connectiontype": 3,
+      "devicetype": 1
+    }
+  },
```

Comment:
I recommend anonymizing the `app` and `device` for tests.

Reply:
Good suggestion, fixed